### PR TITLE
Add iCultus

### PR DESCRIPTION
**Pull request by djyde**

### CODE REVIEW

**Comment by sindresorhus** [comment]
> It's not function integrity app yet.

What does this mean?

-

Please place it at the bottom of the section.

**Comment by sindresorhus** [comment]
Please read: https://github.com/sindresorhus/awesome-electron/blob/master/contributing.md

File: readme.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ Some good apps written with Electron.
 - [LevelUI](https://github.com/hij1nx/levelui) - LevelDB management.
 - [Kart](https://github.com/maddox/kart) - Frontend for RetroArch.
 - [Douban-FM-Express](https://github.com/cyrilis/Douban-FM-Express) - DoubanFM client.
+- [iCultus](https://github.com/djyde/iCultus) - Menubar calendar for OSX.
```
Comment (sindresorhus) [inline readme.md]: It's OS X, not OSX.

**Comment by djyde** [comment]
I'd modified

**Comment by sindresorhus** [comment]
I tried it out and it's a good start, but has a lot of shortcomings. There's currently no way to quit the app. The calendar icon is not retina. The arrows are too small. I should be able to click a date and have it opened in Calendar.app. It should mark days have have calendar events. It does not move correctly when the menubar is set to autohide.

You just released this thing. Try submitting in a month or so when it has had some time to mature ;)

**Pull request closed without merging**